src/screens/LinkContextScreen/elements/Header.js

```diff
@@ -199,77 +185,77 @@ const styles = StyleSheet.create({
     height: 48,
     justifyContent: 'center',
     alignItems: 'center',
-    marginLeft: showBackButton ? 0 : 20,
+    marginLeft: showBackButton ? 0 : 20
   }),
   image: {
     height: 48,
     width: 48,
-    borderRadius: 45,
+    borderRadius: 45
   },
   headerDomainName: {
     fontSize: normalizeFontSize(14),
     fontFamily: fonts.inter[600],
     lineHeight: normalizeFontSize(16.9),
-    color: '#000000',
+    color: COLORS.black
   },
   headerDomainDate: {
     fontFamily: fonts.inter[400],
     fontSize: normalizeFontSize(12),
     lineHeight: normalizeFontSize(18),
-    color: '#828282',
-    flexShrink: 1,
+    color: COLORS.blackgrey,
+    flexShrink: 1
   },
   headerFollowerText: {
-    color: '#828282',
+    color: COLORS.blackgrey,
     fontSize: 12,
-    fontFamily: fonts.inter[700],
+    fontFamily: fonts.inter[700]
   },
   domainItemTitle: {
     fontSize: 16,
     fontFamily: fonts.inter[700],
-    lineHeight: 24,
+    lineHeight: 24
   },
   point: {
     width: 3,
     height: 3,
     borderRadius: 4,
     backgroundColor: COLORS.gray,
     marginLeft: 8,
-    marginRight: 8,
+    marginRight: 8
   },
   domainIndicatorContainer: {
-    marginLeft: -4,
+    marginLeft: -4
   },
   wrapperText: {
-    backgroundColor: 'white',
+    backgroundColor: COLORS.white,
     borderRadius: 8,
-    borderColor: '#00ADB5',
+    borderColor: COLORS.holytosca,
     width: 28,
     height: 28,
     justifyContent: 'center',
     alignItems: 'center',
-    borderWidth: 0.5,
+    borderWidth: 0.5
   },
   wrapperTextUnFollow: {
-    backgroundColor: '#00ADB5',
+    backgroundColor: COLORS.holytosca,
     borderRadius: 8,
-    borderColor: '#00ADB5',
+    borderColor: COLORS.holytosca,
     width: 28,
     height: 28,
     justifyContent: 'center',
     alignItems: 'center',
-    borderWidth: 0.5,
+    borderWidth: 0.5
   },
   leftRowContainer: {
     flex: 1,
     flexDirection: 'row',
-    display: 'flex',
+    display: 'flex'
   },
   backbutton: {
     padding: 16,
     paddingHorizontal: 20,
-    alignSelf: 'center',
-  },
+    alignSelf: 'center'
+  }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There are changes in color constants from hardcoded values to using constants from a theme object. This is a good practice as it improves consistency and maintainability. However, please ensure that these new color constants (`COLORS.white`, `COLORS.black`, `COLORS.blackgrey`, `COLORS.gray`, `COLORS.holytosca`) are defined correctly in the theme object.

```diff
-    color: '#000000',
+    color: COLORS.black

-    color: '#828282',
+    color: COLORS.blackgrey,

-    backgroundColor: 'white',
+    backgroundColor: COLORS.white,

-    borderColor: '#00ADB5',
+    borderColor: COLORS.holytosca,

-    backgroundColor: '#00ADB5',
+    backgroundColor: COLORS.holytosca,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->